src/api-handler.ts

```diff
@@ -12,10 +12,24 @@
 
 'use strict';
 
+import { AddonManagerProxy } from "./addon-manager-proxy";
+
+export interface APIRequestOptions {
+  method: string
+  path: string
+  query: any
+  body: any
+}
```

Comment:
```suggestion
export interface APIRequestOptions {
  method: string
  path: string
  query: {}
  body: {}
}
```

Based on constructor.

Reply:
Done